src/component/veilederverktoy/opprett-oppgave/opprett-oppgave.less

```diff
@@ -87,3 +87,63 @@
         }
     }
 }
+
+// TODO: Dette er en midlertidig workaround mens vi holder på å skrive oss over til nytt designsystem
+// Klassene under overrider styling fra navds-button/navds-label ved at vi "vinner" på CSS-spesifisitet.
+// Grunnen til overridene er at vi per dags dato har en egenutviklet dropdown-komponent.
+// Denne skal byttes ut med Dropdown/Combobox fra nytt designsystem, men inntil videre
+// må vi override noen få styles da vi allerede har tatt i bruk Button fra nytt designsystem.
+.velg-enhet-dropdown {
```

Comment:
Bare et spørsmål her, vi har også en `.velg-enhet-dropdown`-klasse rett ovenfor her (linje 26), er det meningen at det skal være to klasser med samme navn? Det er kanskje uproblematisk/vanlig?